tests/manage/z_cluster/cluster_expansion/test_balance_add_capacity.py

```diff
@@ -0,0 +1,94 @@
+from ocs_ci.ocs.resources.storage_cluster import check_until_osd_ratio_start_decrease_or_equal
+from ocs_ci.utility.retry import retry
+from ocs_ci.ocs.exceptions import ClusterUtilizationNotBalanced
+
+
+logger = logging.getLogger(__name__)
+
+
+@ignore_leftovers
+@tier2
+@polarion_id('OCS-2221')
+@pytest.mark.parametrize("workload_storageutilization_rbd",
+                         [(0.11, True, 5)], indirect=["workload_storageutilization_rbd"])
+class TestRebalaceAddCapacity(ManageTest):
+
+    """
+    Run io till 9% of all cluster, add capacity to the cluster and check the rebalance is initializing
+    """
+
+    def test_rebalance_add_capacity(self, workload_storageutilization_rbd):
+
+        osds_before_expand = []
+        osd_util_dict = get_osd_utilization()
+        logger.info("Checking osds utilization")
+        for osd_name, osd_util in osd_util_dict.items():
+            osds_before_expand.append(osd_name)
+            logger.info(f"osd {osd_name} has {osd_util} util")
+        num_of_osd = len(osds_before_expand)
+        """wait for data to stop increasing"""
+        check_until_osd_ratio_start_decrease_or_equal(osds_before_expand, digit_point=2, ratio_stable=1)
+        logger.info("Adding capacity to the cluster")
+        osd_size = storage_cluster.get_osd_size()
+        result = storage_cluster.add_capacity(osd_size)
+        pod = OCP(
+            kind=constants.POD, namespace=config.ENV_DATA['cluster_namespace']
+        )
+        pod.wait_for_resource(
+            timeout=300,
+            condition=constants.STATUS_RUNNING,
+            selector='app=rook-ceph-osd',
+            resource_count=result * 3
+        )
+
+        pod.wait_for_resource(
+            timeout=300,
+            condition=constants.STATUS_COMPLETED,
+            selector=constants.OSD_PREPARE_APP_LABEL,
+            resource_count=result * 3
+        )
+        num_of_osd_factor = num_of_osd / 3
+        """Waiting for old osd sum of utilization to go down"""
+        check_until_osd_ratio_start_decrease_or_equal(osds_before_expand)
+        check_rebalance_occur_after_expand_with_retry = retry(ClusterUtilizationNotBalanced,
+                                                              tries=(10 * num_of_osd_factor),
+                                                              delay=30, backoff=1)(check_rebalance_occur_after_expand)
+        check_rebalance_occur_after_expand_with_retry(osds_before_expand)
+
+        logger.info("Deleting pod and pvc")
+        ocp = OCP()
+        get_all_pvc_output = ocp.exec_oc_cmd('get pvc -A -o json')
+        for pvc_dict in get_all_pvc_output['items']:
+            logger.info(f"{pvc_dict['metadata']['name']}")
+            if pvc_dict['metadata']['name'] == 'fio-target':
+                logger.info(f"The project name is {pvc_dict['metadata']['namespace']}")
+                test_namespace = pvc_dict['metadata']['namespace']
+
+        from ocs_ci.ocs.utils import get_pod_name_by_pattern
```

Comment:
can this be imported at the top of the file?

Reply:
moved